audit-checklists/Arbitrum.md

```diff
@@ -0,0 +1,95 @@
+# Arbitrum security checklist
+
+## General
+
+### Block number
+
+Invoking `block.number` in a smart contract on Arbitrum will return **L1** block number at which sequencer received transaction, not an **L2** block number. Also worth noting, block number being returned is not necessarily the latest L1 block number. It is the latest synced block number and syncing process occurs approximately every minute. In that period there can be ~5 new L1 blocks. So values returned by `block.number` on Arbitrum do not increase continuously but in "jumps" of ~5 blocks.
```

Comment:
Add a few **the**:

*Arbitrum will return the **L1** block number
*block number at which the sequencer received the transaction
*Also worth noting, the block number being
*and the syncing process occurs

Reply:
Added `the` all over the place hehe